packages/preact/utils/src/index.ts

```diff
@@ -3,10 +3,12 @@ import { useSignal } from "@preact/signals";
 import { Fragment, createElement, JSX } from "preact";
 import { useMemo } from "preact/hooks";
 
+type Falsy = false | 0 | 0n | "" | null | undefined;
```

Comment:
Also falsy: `NaN` but TS doesn't have a type for that specifically, and `document.all`, but I would like to pretend it doesn't exist.